packages/form-js-editor/src/FormEditor.js

```diff
@@ -83,15 +83,23 @@ export default class FormEditor {
   }
 
   destroy() {
+
+    // Destroy form services
     this.get('eventBus').fire('form.destroy');
 
+    // Destroy diagram services (e.g. EventBus)
+    this.get('eventBus').fire('diagram.destroy');
+
     this._detach(false);
   }
 
   /**
    * @param {Schema} schema
    */
   importSchema(schema) {
+    this._emit('diagram.clear');
```

Comment:
Can we move that to a separate `clear` API? That one could clear the registry, too (or the registry hooks up to the clear events, your choice).

Reply:
Done. 👍🏻